Mathlib/Logic/Equiv/Defs.lean

```diff
@@ -522,6 +522,16 @@ noncomputable def propEquivBool : Prop ≃ Bool where
   left_inv p := by simp [@Bool.decide_iff p (Classical.propDecidable _)]
   right_inv b := by cases b <;> simp
 
+/-- `Bool` is equivalent to `Fin 2`. -/
+noncomputable def boolEquivFin : Bool ≃ Fin 2 where
```

Comment:
This already exists, it's `finTwoEquiv`.

Reply:
Of course it's much heavier than `Data.Finite.Defs`, so I've kept my definition, but under the old name